plugins/Microsoft.DocAsCode.Build.UniversalReference/BuildOutputs/ApiBuildOutput.cs

```diff
@@ -25,11 +25,11 @@ public class ApiBuildOutput
 
         [YamlMember(Alias = Constants.PropertyName.Parent)]
         [JsonProperty(Constants.PropertyName.Parent)]
-        public ApiListInDevlangs<ApiNames> Parent { get; set; }
+        public List<ApiLanguageValuePair<ApiNames>> Parent { get; set; }
```

Comment:
List<ApiLanguageValuePair<T> can't be cast to ApiListInDevlangs<T> see: [http://stackoverflow.com/questions/3748931/inheriting-listt-to-implement-collections-a-bad-idea/3748942](http://stackoverflow.com/questions/3748931/inheriting-listt-to-implement-collections-a-bad-idea/3748942)

---
In reply to: [114463310](https://github.com/dotnet/docfx/pull/1618#discussion_r114463310) [](ancestors = 114463310)